github3/repos/repo.py

```diff
@@ -620,6 +620,30 @@ def create_blob(self, content, encoding):
         return sha
 
     @requires_auth
+    def create_branch(self, name, sha=None):
+        """Create a branch. Returns False if branch doesn't exist
+        or not authorized.
+
+        :param str branch: (required), the branch to create.
+        :param str|commit sha: (optional), the commit to base
```

Comment:
Since you're always making an extra API call in the case of sha being None, we need to document that.